tests/test_arcaflow_plugin_pcp.py

```diff
@@ -146,6 +146,46 @@ def test_functional_flat(self):
         self.assertEqual("success", output_id)
         self.assertIsInstance(output_data.pcp_output, list)
 
+    def test_functional_user_pmlogger_conf(self):
+        f = open("tests/pmlogger.conf", "r")
+        pmlogger_conf = f.read()
```

Comment:
I don't see where this file is closed.

Instead of `open()`/`read()`, [as above](https://github.com/arcalot/arcaflow-plugin-pcp/pull/40#discussion_r1565990250) I commend to you `pathlib`:
```python
        pmlogger_conf = Path("tests/pmlogger.conf").read_text()
```

Ditto lines 170-171.